tests/testthat/test-utils.R

```diff
@@ -21,3 +21,19 @@ test_that("sanitize_co2() works with both products and upstream products", {
   out <- sanitize_isic(tibble(input_isic_4digit = 1))
   expect_equal(out$input_isic_4digit, c("1"))
 })
+
+test_that("rename_118() function is applied if issue #118 is addressed", {
+  options(tiltIndicatorAfter.dissable_issue_118 = FALSE)
```

Comment:
In  a test it's best to replace `options()` with `withr::local_options()` -- which automatically cleans up and restores the original options when the test exits.

* [Example usage of `local_options()`](https://github.com/search?q=repo%3A2DegreesInvesting%2FtiltIndicatorAfter+%22local_options%28%22&type=code)
* [`?withr::local_options()`](https://withr.r-lib.org/reference/with_options.html)
* For the general idea of cleaning up see https://testthat.r-lib.org/articles/test-fixtures.html